src/openscad_gui.cc

```diff
@@ -116,8 +116,8 @@ namespace {
 QString assemblePath(const std::filesystem::path& absoluteBaseDir, const std::string& fileName)
 {
   if (fileName.empty()) return "";
-  auto qsDir = QString::fromLocal8Bit(absoluteBaseDir.generic_string().c_str());
-  auto qsFile = QString::fromLocal8Bit(fileName.c_str());
+  auto qsDir = QString::fromStdString(absoluteBaseDir.generic_string());
```

Comment:
I wonder if `genetic_u8string()` should be preferred considering all paths are now created using `u8path()`

Reply:
I just left that as-is without looking at the question.

Looking at it a bit now... generic_u8string() returns std::string until C++20, and std::u8string after that.  Will that change be a compatibility hiccup?

There's a bigger question of whether, in C++20, we should use `std::string` or `std::u8string`, across the program.  My inclination (without understanding the intended distinction) is that `std::u8string` is appropriate for those very few cases where you need to deal with multiple encodings, and that everything is ... cleaner ... if the vast majority of the program does not make encoding assumptions, if encoding is only specified at the periphery.

Reply:
Oh, didn't notice the u8string switch in C++20. Perhaps leave this for now then. Until I read more C++20 codebases, my instinct is to stick with std::string, and use std::u8string where the extra guarantees or disambiguation is valuable.